README.md

```diff
@@ -255,7 +255,7 @@ curl www.bar-app.docker/api/v1/baz_service/qzz
 
 ### dip dns
 
-Runs DNS server container based on https://github.com/aacebedo/dnsdock It used for container to container requests through nginx. An application's docker-compose.yml should define `dns` configuration with environment variable `$DIP_DNS` and connect to external network `frontend`. `$DIP_DNS` will be automatically assigned by dip.
+It runs a DNS server container based on https://github.com/aacebedo/dnsdock. It is used for container to container requests through Nginx. An application's docker-compose.yml should define `DNS` configuration with environment variable `$DIP_DNS` and connect to external network `frontend`. `$DIP_DNS` will be automatically assigned by dip.
```

Comment:
I think there is no need `It` at the beginning

Reply:
If I remove the `It` Grammarly complain.
<img width="468" alt="Capture d’écran, le 2019-08-13 à 16 39 59" src="https://user-images.githubusercontent.com/3338913/62923617-19d61180-bde9-11e9-823d-512ef32fb3c9.png">

Reply:
Robots can also be wrong :) The above sentences without `It` too and it doesn't complain.

Reply:
I'll remove it :) (I don't really have a strong opinion)